src/applications/personalization/profile/tests/fixtures/users/user-not-in-mpi.json

```diff
@@ -3,18 +3,7 @@
     "id": "",
     "type": "users_scaffolds",
     "attributes": {
-      "services": [
```

Comment:
Not at all related to the main point of this PR, but I'm going to start cleaning up mock users when I see them with lots of unnecessary properties.